firmware/src/tasks/task_telemetry.hpp

```diff
@@ -40,23 +41,38 @@ class Telemetry final : public Task<Telemetry, 1024> {
     int16_t velocity : 10;
     uint16_t voltage : 8;
     uint16_t pyro_continuity : 2;
+    uint8_t testing_on : 1;
     // fill up to 16 bytes
     uint8_t : 0;  // sent
     uint8_t d1;   // dummy
   } __attribute__((packed));
 
+  struct packed_rx_msg_t {
+    uint8_t header;
+    uint32_t passcode;
+    uint8_t event;
+    uint8_t enable_testing_telemetry;
+    uint32_t dummy1;
+    uint32_t dummy2;
+  } __attribute__((packed));
+
+  uint32_t m_uplink_phrase_crc = 0;
+  bool m_testing_enabled;
+  bool m_enable_testing_telemetry = false;
```

Comment:
Maybe rename to something like:
```suggestion
  bool m_testing_armed = false;
```